toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/WorldScaleTrackingMode.kt

```diff
@@ -19,18 +19,31 @@
 package com.arcgismaps.toolkit.ar
 
 /**
- * Used to determine how to control the camera in a [WorldScaleSceneView].
+ * Defines how the position and orientation of a device are determined and how the
+ * [WorldScaleSceneView]'s camera is synchronized with that position and orientation.
  *
  * @since 200.7.0
  */
 public sealed class WorldScaleTrackingMode {
     /**
-     * The camera will be controlled using [Google's Geospatial API](https://developers.google.com/ar/develop/geospatial).
+     * The camera is controlled using [Google's ARCore Geospatial API](https://developers.google.com/ar/develop/geospatial).
+     * This mode uses a combination of [VPS](https://developers.google.com/ar/develop/geospatial#global_localization_with_vps)
+     * and GPS to determine the position and orientation of the device.
+     *
+     * This mode requires [authorization with the ARCore service or an ARCore API key](https://developers.google.com/ar/develop/authorization?platform=android).
+     * Usage is constrained by [Google's API usage quota](https://developers.google.com/ar/develop/java/geospatial/api-usage-quota).
+     *
+     * @since 200.7.0
      */
     public class Geospatial : WorldScaleTrackingMode()
 
     /**
-     * The camera will be controlled using a location data source and the device sensors.
+     * The camera is controlled using a combination of GPS for device location and ARCore for device
+     * orientation. Tracking accuracy depends on the accuracy of the GPS signal. Manual calibration
```

Comment:
Nitpick, since we also use the fused orientation provider
```suggestion
     * The camera is controlled using a combination of GPS for device location, and the device sensors and ARCore for device
     * orientation. Tracking accuracy depends on the accuracy of the GPS signal. Manual calibration
```